cfgov/v1/jinja2/v1/includes/molecules/hero.html

```diff
@@ -62,7 +66,11 @@
                 {{- ' m-hero__jumbo' if value.is_jumbo else '' }}">
     <div class="m-hero_wrapper">
         <div class="m-hero_text">
-            <h1 class="m-hero_heading">{{ value.heading | safe }}</h1>
+            <h1 class="m-hero_heading">{{ value.heading | safe }}
+                {% if value.heading_continued %}
+                <br> {{ value.heading_continued }}
+                {% endif %}p
```

Comment:
There's an errant `p` here it looks like

```suggestion
                {% endif %}
```